lib/simplecov/source_file.rb

```diff
@@ -149,46 +246,27 @@ def relevant_lines
       lines.size - never_lines.size - skipped_lines.size
     end
 
-    # Returns all covered lines as SimpleCov::SourceFile::Line
-    def covered_lines
-      @covered_lines ||= lines.select(&:covered?)
+    def no_branches?
+      total_branches.length.zero?
     end
 
-    # Returns all lines that should have been, but were not covered
-    # as instances of SimpleCov::SourceFile::Line
-    def missed_lines
-      @missed_lines ||= lines.select(&:missed?)
-    end
+    def branches_coverage_precent
```

Comment:
Shouldn't it be `branches_coverage_percent `?

Reply:
Updated! Just in case, some of these methods are used in the default formatter and needs to be changed in both places.